hw/top_earlgrey/data/ip/chip_otp_ctrl_testplan.hjson

```diff
@@ -0,0 +1,208 @@
+      features: ["OTP_CTRL.ENTROPY_READ"]
+      stage: V2
+      si_stage: SV1
+      tests: ["chip_sw_sram_ctrl_scrambled_access",
+              "chip_sw_flash_init",
+              "chip_sw_keymgr_key_derivation",
+              "chip_sw_otbn_mem_scramble",
+              "chip_sw_rv_core_ibex_icache_invalidate"]
+    }
+    {
+      name: chip_sw_otp_ctrl_program
+      desc: '''Verify the program request from lc_ctrl.
+
+            - Verify that upon an LC state transition request, LC ctrl signals the OTP ctrl with a
+              program request.
+            - Verify that the OTP ctrl generates the LC data output correctly and is sent to the LC
+              ctrl before it is reset.
+            - Verify that the `lc_check_byp_en_i` from LC ctrl is set.
+            - Ensure that the whole operation does not raise any interrupts or alerts or errors.
+            - After reset, verify that the LC state transition completed successfully by reading the
+              LC state and LC count CSRs.
+            '''
+      features: ["OTP_CTRL.PROGRAM", "OTP_CTRL.PARTITION.SECRET0",
+                 "OTP_CTRL.PARTITION.LIFE_CYCLE", "OTP_CTRL.PARTITION.SECRET2"]
+      stage: V2
+      si_stage: SV1
+      tests: ["chip_sw_lc_ctrl_transition"]
+    }
+    {
+      name: chip_sw_otp_ctrl_program_error
+      desc: '''Verify the otp program error.
+
+            - Initiate an illegal program request from LC ctrl to OTP ctrl by forcing the
+              `lc_otp_program_i`.
+            - Verify that the LC ctrl triggers an alert and escalation when the OTP ctrl responds back with the correct
+              fatal program error as expected.
+            - Allow escalation to go through reset.
+            - Use the rstmgr alert info and the fault CSRs to check the alert cause is right.
+            - Keep state across resets in flash to check that no NMI or regular ISR interrupt run
+              for kTopEarlgreyAlertIdLcCtrlFatalProgError.
+            - Verify that, after escalation reset, the alert handler cause and the fault register are cleared.
+            - Verify after the escalation reset that the alert info captured the correct alert.
+            '''
+      features: ["OTP_CTRL.ERROR_HANDLING"]
+      stage: V2
+      si_stage: SV3
+      tests: ["chip_sw_lc_ctrl_program_error"]
+    }
+    {
+      name: chip_sw_otp_ctrl_hw_cfg
+      desc: '''Verify the correctness of otp_hw_cfg bus in all peripherals that receive it.
+
+            Preload the OTP ctrl's `hw_cfg` partition with random data and verify that all
+            consumers of the hardware configuration bits are receiving the correct values.
+
+            Xref'ed with corresponding IP tests that receive these bits.
+            '''
+      features: ["OTP_CTRL.ENTROPY_READ"]
+      stage: V2
+      si_stage: SV1
+      tests: ["chip_sw_lc_ctrl_otp_hw_cfg"]
+    }
+    {
+      name: chip_sw_otp_ctrl_lc_signals
+      desc: '''Verify the broadcast signals from LC ctrl.
```

Comment:
In hardware lots of these tests are not feasible. Please add a `SiVal` comment explaining what can be tested. Also, some of this functionality is required for success of the early manufacturing tests, so add these tests below.

Reply:
Then please state the checks without referring to specific signals, since they are not observable. We can only check correctness based on things like life cycle states.

Reply:
I converted to si_stage: None for some tests cannot be verified with current testpoints.